mesoshttp/client.py

```diff
@@ -39,13 +42,13 @@ class SchedulerDriver(CoreMesosObject):
         Handler to communicate with scheduler
 
         `MesosClient.SchedulerDriver` instance is available after the
-        SUBSCRIBED event with the subcribed event.
+        SUBSCRIBED event with the subscribed event.
         '''
-        def __init__(self, mesos_url, frameworkId, streamId):
+        def __init__(self, mesos_url, frameworkId, streamId, requests_auth, verify):
```

Comment:
same issue as other init you fixed , for backward compatibility, fields should be optional with defaults when creating a SchedulerDriver object